plugins/woocommerce/client/legacy/js/admin/wc-shipping-zone-methods.js

```diff
@@ -388,11 +389,12 @@
 								}
 								// Trigger save if there are changes, or just re-render
```

Comment:
Hmm although these changes do prevent the **Save changes** from being disabled again, it is actually the request above that saves the data. So even if I don't hit **Save changes** and refresh the page the new shipping method is still in the list.